images/build/Dockerfiles/gitHubActions.Dockerfile

```diff
@@ -118,6 +118,17 @@ COPY --from=nodetools-install /opt /opt
 COPY --from=buildscriptgenerator /opt/buildscriptgen/ /opt/buildscriptgen/
 RUN ln -s /opt/buildscriptgen/GenerateBuildScript /opt/oryx/oryx
 
+# Install PHP pre-reqs
+RUN ${IMAGES_DIR}/build/php/prereqs/installPrereqs.sh
+# NOTE: do not include the following lines in prereq installation script as
+# doing so is causing different version of libargon library being installed
```

Comment:
do we know why that is?

Reply:
this is causing a pain for buster images as well... in the installprereq script it seems like we are updating the apt source list for libargon to install certain version, but there is no documentation which currently explains it.
##<argon2>##
sed -e 's/stretch/buster/g' /etc/apt/sources.list > /etc/apt/sources.list.d/buster.list;
{ \
	echo 'Package: *';
	echo 'Pin: release n=buster';
	echo 'Pin-Priority: -10';
	echo;
	echo 'Package: libargon2*';
	echo 'Pin: release n=buster';
	echo 'Pin-Priority: 990';
} > /etc/apt/preferences.d/argon2-buster;
apt-get update;
apt-get install -y --no-install-recommends libsodium-dev;
##</argon2>##